raiden-dapp/src/views/TransferSteps.vue

```diff
@@ -353,6 +378,13 @@ export default class TransferSteps extends Mixins(
             path
           } as Route)
       );
+
+      const [route] = this.routes;
+
+      if (route && route.fee.isZero()) {
+        this.selectedRoute = route;
+        this.transfer();
```

Comment:
Maybe go to step 3 first and let the user confirm the transfer. Otherwise from UX he would see the 3 steps at the top and then move directly from 2nd step to the transfer screen. Very minor

Reply:
We already discussed with @taleldayekh that this flow can get improved. The plan is to evaluate all the possible scenarios and figure out the proper flow.